lib/plugins/basalprofile.js

```diff
@@ -57,8 +62,18 @@ function init() {
       info.push({label: 'Active temp basal remaining', value: remaining + ' min'});
     }
       
+    if (basalValue.combobolustreatment) {
+      tempText = (basalValue.combobolustreatment.relative ? '+' + basalValue.combobolustreatment.relative + 'U/h' : '');
+      remaining = parseInt(basalValue.combobolustreatment.duration - times.msecs(sbx.time - basalValue.combobolustreatment.mills).mins);
+      info.push({label: '------------', value: ''});
+      info.push({label: 'Active combo bolus', value: tempText});
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['tempText' used out of scope.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1498320423/issues/source?bid=2452379&fileBranchId=2998783#l69)